include/GafferScene/MeshTangents.h

```diff
@@ -79,6 +79,14 @@ class GAFFERSCENE_API MeshTangents : public SceneElementProcessor
 
 		IE_CORE_DECLARERUNTIMETYPEDEXTENSION( GafferScene::MeshTangents, MeshTangentsTypeId, SceneElementProcessor );
 
+    enum Mode
```

Comment:
Tabs not spaces. Let's also put it higher up, so the reader has bumped into it before hitting the `modePlug()` - just one more clue what it's for.

Reply:
Moved it up and converted to tabs. (...emacs hates me, and once in a while disables the c++ formatting layer) :D